scipy/stats/__init__.py

```diff
@@ -265,76 +266,165 @@
+coordinates of multivariate observations.
+
+.. autosummary::
+   :toctree: generated/
+
+   linregress
    pearsonr
    spearmanr
    pointbiserialr
    kendalltau
    weightedtau
    somersd
-   linregress
    siegelslopes
    theilslopes
+   page_trend_test
    multiscale_graphcorr
 
-Statistical tests
-=================
+These association tests and are to work with samples in the form of contingency
+tables. Supporting functions are available in `scipy.stats.contingency`.
+
+.. autosummary::
+   :toctree: generated/
+
+   chi2_contingency
+   fisher_exact
+   barnard_exact
+   boschloo_exact
+
+Independent Sample Tests
+------------------------
+Independent sample tests are typically used to assess whether multiple samples
+were independently drawn from the same distribution or different distributions
+with a shared property (e.g. equal means).
+
+Some tests are specifically for comparing two samples.
 
 .. autosummary::
    :toctree: generated/
 
-   ttest_1samp
-   ttest_ind
    ttest_ind_from_stats
-   ttest_rel
-   chisquare
-   cramervonmises
+   poisson_means_test
+   ttest_ind
+   mannwhitneyu
+   brunnermunzel
+   mood
+   ansari
    cramervonmises_2samp
-   power_divergence
-   kstest
-   ks_1samp
-   ks_2samp
    epps_singleton_2samp
-   mannwhitneyu
-   tiecorrect
-   rankdata
-   ranksums
-   wilcoxon
+   ks_2samp
+   kstest
+
+Others are generalized to multiple samples.
+
+.. autosummary::
+   :toctree: generated/
+
+   f_oneway
+   tukey_hsd
    kruskal
+   alexandergovern
+   fligner
+   levene
+   bartlett
+   median_test
    friedmanchisquare
-   brunnermunzel
+   anderson_ksamp
+
+Resampling and Monte Carlo Methods
+----------------------------------
+The following functions can reproduce the p-value and confidence interval
+results of most of the functions above, and often produce accurate results in a
+wider variety of conditions. They can also be used to perform hypothesis tests
+and generate confidence intervals for custom statistics. This flexibility comes
+at the cost of greater computational requirements and stochastic results.
+
+.. autosummary::
+   :toctree: generated/
+
+   monte_carlo_test
+   permutation_test
+   bootstrap
+
+Multiple Hypothesis Testing and Meta-Analysis
+---------------------------------------------
+These functions are for assessing the results of individual tests as a whole.
+Functions for performing specific multiple hypothesis tests (e.g. post hoc
+tests) are listed above.
+
+.. autosummary::
+   :toctree: generated/
+
    combine_pvalues
-   jarque_bera
-   page_trend_test
-   tukey_hsd
-   poisson_means_test
+
+Deprecated and Legacy Functions
+-------------------------------
 
 .. autosummary::
    :toctree: generated/
 
-   ansari
-   bartlett
-   levene
-   shapiro
-   anderson
-   anderson_ksamp
    binom_test
-   binomtest
-   fligner
-   median_test
-   mood
-   skewtest
-   kurtosistest
-   normaltest
-   goodness_of_fit
+   tiecorrect
+   ranksums
```

Comment:
```suggestion
```
I will pursue getting these deprecated, but I'll move them elsewhere for now so we can merge this.